docker-compose-prod.yml

```diff
@@ -8,6 +8,9 @@ x-volumes: &default-volume
 x-restart-policy: &restart_policy
     restart: unless-stopped
 
+x-restart-policy-always: &restart_policy_always
```

Comment:
All containers must have the same restart policy. For example, after Docker daemon restarts, database is not started automatically, but worker is require a database